purchase_order_move_menu_pot/models/stock_move.py

```diff
@@ -0,0 +1,16 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import fields, models, api
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    po_menu_exclude = fields.Boolean(related='purchase_order.order_type.stock_move_menu_exclude', string='Menu Exclude', store=True)
```

Comment:
Use a more specific label:
```suggestion
    po_menu_exclude = fields.Boolean(related='purchase_order.order_type.stock_move_menu_exclude', string='Excluded for PO moves menu', store=True)
```